codice/ddf
# DDF-2070 Add default config files for ldap features

#### What does this PR do?
Add config files for the ldap features (security-sts-ldaplogin, security-sts-ldapclaimshandler) so that they are configured by default after being installed.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@ryeats @bcwaters @tbatie @coyotesqrl @jckilmer 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire
@stustison
#### How should this be tested?
-Install security-sts-ldaplogin and security-sts-ldapclaimshandler featureas and verify they are configured after being installed. 
-Verify logging in with an LDAP server connected still works.
#### Any background context you want to provide?
After the merge of DDF-2033, these feature are not configured by default anymore.
#### What are the relevant tickets?
DDF-2070
#### Screenshots (if appropriate)
#### Checklist:
